packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -426,7 +426,7 @@ class SnackBar extends StatefulWidget {
   /// than this percentage of the width of the snackbar less the width of its [action],
   /// then the [action] will appear below the [content].
   ///
-  /// At a value of 0, the action will not overflow to a new line.
+  /// At a value of 1, the action will not overflow to a new line.
```

Comment:
This does not seem to match the description above.
Maybe this comment was "At a value of 0, the action will always overflow to a new line."

Can you check with various cases? For instance with a content which is way longer than the snackbar width? In that case, even with a value of 1, the condition described in the existing comment should still stand, no?
" /// If the width of the snackbar's [content] is greater
  /// than this percentage of the width of the snackbar less the width of its [action],
  /// then the [action] will appear below the [content]."

Maybe the whole comment is wrong. We should check the bahavior and have a look at the implementation.

Reply:
@bleroux  I was exploring using 0 and 1 while updating the view width and this was the result:

`actionOverflowThreshold = 0`
https://github.com/user-attachments/assets/7d1e36b7-fbcf-4020-9d97-272eac5bfd4a

`actionOverflowThreshold = 1`
https://github.com/user-attachments/assets/5f70a33b-37c4-428d-9a7b-e53e23768dcf


I do believe "At a value of 0, the action will always overflow to a new line." is the best phrase.

Reply:
@bleroux 
I double-checked the behavior when the value is set to 1, this time testing with different content sizes, and I found a mistake. I believe this is an edge case, but even when the value is 1, the action can still overflow to a new line.

That said. Can we keep the original comment and just update the phrase about the 0 value?
Like:

```
/// If the width of the snackbar's [content] is greater
/// than this percentage of the width of the snackbar less the width of its [action],
/// then the [action] will appear below the [content].
///
/// At a value of 0, the action will always overflow to a new line.
```


 
![Screenshot 2024-10-17 at 18 53 42](https://github.com/user-attachments/assets/be593cf7-85cd-43e4-bc63-d059747a5fe3)

Reply:
Interesting, reading the code it is possible to get a percentage greater than 1 when the action width is greater that the snackbar width (that seems to be the case on your screenshot, where action label is 'Action Action Action'.

I would propose to go with the following:

```dart
/// If the width of the snackbar's [action] divided by the width of the snackbar
/// is greater than this percentage then the [action] will appear below the [content].
///
/// At a value of 0, the action will always overflow to a new line. 
```

Unless your more comfortable with the original comment.

Reply:
I updated it to follow your suggestion!